src/components/RadioInputBox/DescriptionSummary.tsx

```diff
@@ -2,9 +2,10 @@ type Props = {
   description: string;
 };
 
-export default function DescriptionSummary({
-  description,
-}: Props): JSX.Element {
+export default function DescriptionSummary({ description }: Props) {
```

Comment:
The `Props` type should be updated to include the falsy types that `description` can be. In MAAS we typically refer to the actual API type itself (e.g. `Machine["name"]` instead of `string`) so that any changes to the shape of the API are known by all the components that consume it.

Reply:
Oops, that'll teach me to be distracted :) Updated the type.

Yeah the API structure doesn't change, but what I want to do is import the types from libjuju just haven't yet had the time yet.